dockers/docker-snmp-sv2/config.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+if dpkg-query -W --showformat='${Version}' snmp |grep -q "5.7.2"; then
+  snmp_user=snmp
+else
+  snmp_uesr=Debian-snmp
+fi
```

Comment:
I think we do not need this "if" as the snmp version is embedded in the buildimage, we know which snmp version we are using.

just use snmp_user=snmp